fv3core/fv3core/stencils/remapping.py

```diff
@@ -1,5 +1,7 @@
 from typing import Dict
 
+# [DaCe] import
+from dace import constant as dace_constant
```

Comment:
@FlorianDeconinck I'm not sure how you feel about this, but my opinion has been that types should not be formatted like this. I would suggest `DaceConstant` is a type, `dace_constant` is an instance.

Reply:
'tis more pythonic. Ultimately those will disappear altogether.